### PR TITLE
Prepare codebase for hand-off

### PR DESCRIPTION
**Pull request by willemolding**
Add last few details before shipping the additions:

- [x] Add license file
- [x] Add inline docs where appropriate
- [x] Add small tutorial for consuming code
- [ ] Testing guide ("Core functions will be fully covered by unit tests and e2e tests using polywrap recipes json tests")
- [x]  Clean up repo TODO list

### CODE REVIEW

File: protocol/substrate/rpc-wrapper/README.md [hunk 1]
```diff
@@ -1,22 +1,82 @@
...
-- [ ] Complete test suite to include failure cases
-
 ## Prerequisite
 - Install a recent version of node
     `nvm install & nvm use`
+- Docker installed (see OS specific installation guides)
+
+## Using the Wrapper
+
+The Substrate core wrapper can either be used directly in an application or used to make dApp specific wrapper.
```
Comment (empea-careercriminal) [inline protocol/substrate/rpc-wrapper/README.md:12]: typo: a dApp specific ...

File: protocol/substrate/rpc-wrapper/README.md [hunk 2]
```diff
@@ -1,22 +1,82 @@
...
+
+### Using directly (js example)
+
+The wrapper requires a Substrate signer-provider in order to sign transactions. Current the only implementation for this is for a browser javascript host and integrates with the polkadot-js browser extension for signing. In the future other signing providers might be added.
```
Comment (empea-careercriminal) [inline protocol/substrate/rpc-wrapper/README.md:16]: typo: current --> currently

File: protocol/substrate/rpc-wrapper/schema.graphql [hunk 3]
```diff
@@ -3,38 +3,100 @@
...
+  pallet: Name of pallet
+  storage: Name of storage slot
+  """
   getStorageValue(url: String!, pallet: String!, storage: String!): [UInt8!]
 
+  """
+  Return a value in a storage map
+  pallet: Name of pallet
+  storage: Name of storage map
+  key: key into storage map to retrieve
```
Comment (empea-careercriminal) [inline protocol/substrate/rpc-wrapper/schema.graphql:54]: typo: key --> Key

File: protocol/substrate/signer-provider-js/src/index.ts [hunk 4]
```diff
@@ -27,6 +27,10 @@ export class SubstrateSignerProviderPlugin extends Module<SubstrateSignerProvide
     this._registry = new TypeRegistry();
   }
 
+  /**
+   * Return the accounts managed by the connected polkadot-js plugin.
+   * args is an empty arg just included to meet the function signature requirementss
```
Comment (empea-careercriminal) [inline protocol/substrate/signer-provider-js/src/index.ts:32]: typo: requirementss -> requirements

**Review by boorich — CHANGES_REQUESTED** [review]
@willemolding Just being pedantic there are some typos in the documentation where I left suggestions.

File: protocol/substrate/rpc-wrapper/README.md (hunk 1 above)
Comment (boorich) [inline protocol/substrate/rpc-wrapper/README.md]:
```suggestion
The Substrate core wrapper can either be used directly in an application or used to make a dApp specific wrapper.
```

File: protocol/substrate/rpc-wrapper/README.md (hunk 2 above)
Comment (boorich) [inline protocol/substrate/rpc-wrapper/README.md]:
```suggestion
The wrapper requires a Substrate signer-provider in order to sign transactions. Currently the only implementation for this is for a browser javascript host and integrates with the polkadot-js browser extension for signing. In the future other signing providers might be added.
```

File: protocol/substrate/rpc-wrapper/schema.graphql [hunk 5]
```diff
@@ -3,38 +3,100 @@
...
+  """
   genesisHash(url: String!): String
 
+  """
+  Return a block given its number/height
+  """
   chainGetBlock(url: String!, number: UInt32!): BlockOutput
 
+  """
+  Retrieve a constant valud from a pallet in the connected chain
```
Comment (boorich) [inline protocol/substrate/rpc-wrapper/schema.graphql]:
```suggestion
  Retrieve a constant value from a pallet in the connected chain
```

File: protocol/substrate/rpc-wrapper/schema.graphql [hunk 6]
```diff
@@ -3,38 +3,100 @@
...
+  pallet: Name of pallet
+  constant: name of constant
```
Comment (boorich) [inline protocol/substrate/rpc-wrapper/schema.graphql]:
```suggestion
  constant: Name of constant
```

File: protocol/substrate/rpc-wrapper/schema.graphql (hunk 3 above)
Comment (boorich) [inline protocol/substrate/rpc-wrapper/schema.graphql]:
```suggestion
  key: Key into storage map to retrieve
```

File: protocol/substrate/signer-provider-js/src/index.ts (hunk 4 above)
Comment (boorich) [inline protocol/substrate/signer-provider-js/src/index.ts]:
```suggestion
   * args is an empty arg just included to meet the function signature requirements
```

**Pull request merged**